modules/meteroid/migrations/diesel/2025-01-20-000001_bank_accounts/up.sql

```diff
@@ -0,0 +1,24 @@
+ 
+create type "BankAccountFormat" as enum ('IBAN_BIC_SWIFT', 'ACCOUNT_ROUTING', 'SORT_CODE_ACCOUNT', 'ACCOUNT_BIC_SWIFT');
+ 
+CREATE TABLE "bank_account"
+(
+  "id"              UUID                                NOT NULL PRIMARY KEY,
+  "local_id"        TEXT                                NOT NULL,
+  "tenant_id"       UUID                                NOT NULL REFERENCES "tenant" ON DELETE RESTRICT,
+  "currency"        TEXT                                NOT NULL,
+  "country"         TEXT                                NOT NULL,
+  "bank_name"       TEXT                                NOT NULL,
+  "format"          "BankAccountFormat"                 NOT NULL,
+  "account_numbers" TEXT                                NOT NULL,
```

Comment:
hm, is TEXT ok for this? probably array would fit better in case we want to index and do search?

Reply:
decided to go with text + ilike for search